database/gormdb_group_test.go

```diff
@@ -202,12 +202,6 @@ func TestGormDBCreateAndGetGroup(t *testing.T) {
 				}
 			}
 			group.Enrollments = enrollments
-			for _, usr := range have.Users {
-				usr.Enrollments = nil
-			}
-			for _, e := range have.Enrollments {
-				e.User.Enrollments = nil
-			}
```

Comment:
Let me know if tests are sufficient already.

Reply:
I believe `TestGormDBCreateAndGetGroup` and `TestGetGroupsByCourse` are sufficient already.